src/libraries/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -45,6 +45,7 @@ public SafeDeleteNegoContext(SafeFreeNegoCredentials credential)
             : base(credential)
         {
             Debug.Assert((null != credential), "Null credential in SafeDeleteNegoContext");
+            _context = new SafeGssContextHandle();
```

Comment:
Instead of trying to handle `null` values here it's easier to just always have a non-null handle.

Reply:
I assume we need to always allocate it anyway? #69527 is trying to avoid unnecessary allocations.